### PR TITLE
Pass 2: Methods.

### PR DESCRIPTION
**Pull request by costdev**

### CODE REVIEW

File: src/wp-admin/includes/class-rollback-auto-update.php [hunk 1]
```diff
@@ -422,84 +422,78 @@ private function restart_core_updates() {
...
-
 		// Get array of plugins set for auto-updating.
 		$auto_updates    = (array) get_site_option( 'auto_update_plugins', array() );
 		$current_plugins = array_keys( self::$plugin_updates->response );
 
 		// Get all auto-updating plugins that have updates available.
 		$current_auto_updates = array_intersect( $auto_updates, $current_plugins );
 
-		// Get array of non-fatal auto-updates remaining.
+		// Get array of remaining auto-updates, excluding plugins that were rolled back.
```
Comment (afragen) [inline src/wp-admin/includes/class-rollback-auto-update.php]: excluding plugins processed or rolled back.

File: src/wp-admin/includes/class-rollback-auto-update.php [hunk 2]
```diff
@@ -422,84 +422,78 @@ private function restart_core_updates() {
...
-	private function restart_updates_and_send_email() {
-		$this->restart_updates();
-		$this->restart_core_updates();
+	private function resume_updates_and_send_email() {
+		$this->resume_updates();
+		$this->resume_core_update();
 
 		/*
 		 * The following commands only run once after the above commands have completed.
 		 * Specifically, 'restart_updates()' will re-run until there are no further
```
Comment (afragen) [inline src/wp-admin/includes/class-rollback-auto-update.php]: update comment for new method name.

File: src/wp-admin/includes/class-rollback-auto-update.php (hunk 1 above)
Comment (costdev) [reply src/wp-admin/includes/class-rollback-auto-update.php]: Fixed ✅

File: src/wp-admin/includes/class-rollback-auto-update.php (hunk 2 above)
Comment (costdev) [reply src/wp-admin/includes/class-rollback-auto-update.php]: Fixed ✅

**Pull request merged**